intranet/static/themes/piday/piday.js

```diff
@@ -3,5 +3,5 @@ $(function() {
         "</marquee>"
-    ).appendTo($("body"));
-});
+    ).appendTo(document.body);
+});
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://www.codacy.com/app/tjcsl/ion/file/2769584683/issues/source?bid=3176160&fileBranchId=3355598#l7)